amical/externals/pymask/__init__.py

```diff
@@ -49,13 +46,11 @@
 # import x_y_plot
 from .x_y_plot import *
 
+
+# TODO(clm): move this somewhere controlable
+# (plt.rcParams)["image.origin"] = "lower"
+# (plt.rcParams)["image.interpolation"] = "nearest"
 # -------------------------------------------------
 # set some defaults to display images that will
 # look more like the DS9 display....
 # -------------------------------------------------
-# plt.set_cmap(cm.gray)
-(plt.rcParams)["image.origin"] = "lower"
-(plt.rcParams)["image.interpolation"] = "nearest"
```

Comment:
probably not backward compatible. Needs discussion

Reply:
after discussion with @DrSoulain I am now convinced that this wasn't *intentionally* baked into AMICAL. I'll just drop it.